src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -0,0 +1,169 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.github.lindenb.jbwa.jni.AlnRgn;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.CigarUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.List;
+import java.util.Objects;
+
+class AlignmentRegion {
+
+    final String contigId;
+    final String breakpointId;
```

Comment:
This issue with that is that we are storing `AlignmentRegion` objects and reading them in (in another tool) before converting them into `AssembledBreakpoint` objects. So I need to denormalize the ID for that reason.